packages/backend/src/layers/functionality/statement/get-monobank-statement.service.ts

```diff
@@ -45,36 +54,90 @@ const toInTimestamp = () => {
 export class GetMonobankStatementService {
   constructor(
     private monobankIntegration: MonobankIntegration,
-    private statementStorage: StatementStorage,
     private configService: ConfigService,
+    private statementStorage: StatementStorage,
+    private featureFlagStorage: FeatureFlagStorage,
+    private importAttemptStorage: ImportAttemptStorage,
   ) {}
 
-  async getStatement({ tokenId }: IGetStatement) {
-    const accountList = await this.statementStorage.getAccountByTokenId(
-      tokenId,
+  async getStatement({ tokenId, importAttemptId }: IGetStatement) {
+    console.log(tokenId, importAttemptId);
+    const featureFlag = await this.featureFlagStorage.getFeatureFlag(
```

Comment:
I suggest to discuss the API of feature flags service

Reply:
Example of improvement:

```
const featureFlags = await this.featureFlagStorage.getFeatureFlags();

if (featureFlags[FeatureName.BypassMonobankRateLimit]) {

}
```

Reply:
Done